### DISCUSSION TITLE
ender sprite direct extruder settings

### DISCUSSION THREAD

**Discussion by RonJongbloed** [mriscoc/Ender3V2S1#205] [General]
i installed a ender sprite direct extruder on my ender 3v2. installed the ender3v2-422blt-am3960 firmware, my problem is that the probe goes further that the bed how can i correct this

thanks
Ron

**Comment by Nbrooks639** [comment]
most likely nozzle to probe offsets are incorrect now, you should be able to fix under probe settings.

**Comment by mriscoc** [comment]
Please check: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Comment by steverimbey** [comment]
I put the Sprite + CR Touch on my heavily modified Ender 3 Pro. And had same problem... CR Touch probe was below the nozzle and interfered with the print.

Fixed by gently lifting the sprite before tightening the mounting screws. You will find that there is enough "slop" in the mounting that the extruder can be pivoted up enough to eliminate the interference.
